src/__tests__/TuxActions-test.js

```diff
@@ -87,9 +87,105 @@ describe('TuxActions', function () {
 
         tuxActionCategory.register(mockStore, actionListeners);
 
-        expect(TuxActions.register.mock.calls[0][0]).toEqual(mockStore);
+        expect(TuxActions.register.mock.calls[0][0]).toBe(mockStore);
         //expecting to find actionListeners under the CATEGORY key
-        expect(TuxActions.register.mock.calls[0][1].tests).toEqual(actionListeners);
+        expect(TuxActions.register.mock.calls[0][1].tests).toBe(actionListeners);
+      });
+
+      describe('before', function () {
+        it('should add a callback to be invoked before the corresponding action is dispatched and should receive the next callback to invoke and the passed in actionBody as inputs', function () {
```

Comment:
Recommend changing "...to invoke and the passed in actionBody as inputs" to "...to invoke with the passed in actionBody as inputs".